src/util.cpp

```diff
@@ -568,6 +568,13 @@ int64_t GetArg(const std::string &strArg, int64_t nDefault)
     return nDefault;
 }
 
+double GetDoubleArg(const std::string &strArg, double dDefault)
+{
+    if (mapArgs.count(strArg))
+        return atof(mapArgs[strArg].c_str()); // returns 0.0 on conversion failure
```

Comment:
Value on conversion failure is consistent with `int64_t GetArg`. Hiding a conversion error and just returning some value is a bad convention though, but out of scope for this PR I think.

Reply:
I agree -- the preferred pattern I usually like is passing an optional `bool * = nullptr` arg and if that's present, writing the success/error result of the conversion there. If you like I can add a PR after this one that does that for all the `GetArg` functions in this codebase.. it would be easy/fun to do so... :)

Reply:
That's a good proposal and I would definitely support it (I don't have merge privileges though, so can't promise anything).

Reply:
I'll merge that PR!

Reply:
... or throw an exception? ...your choice